### ISSUE TITLE
Find manga from all already indexed/downloaded manga site/connector

### ISSUE THREAD

**Issue by kokizzu** [manga-download/hakuneko#1114]
Add one more type other than "bookmarked"/"folder view"/"clipboard", that is "indexed" for all sites that already being indexed.
![image](https://user-images.githubusercontent.com/1061610/74509557-63d4b700-4f34-11ea-9a3e-a07ce9c0f961.png)

**Comment by 27136773** [comment]
Please provide more details about what shall be achieved.

Currently it seems the request is to add a new connector that will show all mangas from all websites that has been synced, therefore the connector would show a list of multiple thousands of mangas when selected?

Or shall there be a tag to hide all websites that are not yet synced from the website selection list?

**Comment by kokizzu** [comment]
yes, hundreds of thousands of mangas at once, just like bookmark, but with [suffix] of the connector

**Comment by kokizzu** [comment]
for the tag to hide all websites that are not yet synced will be good solution for https://github.com/manga-download/hakuneko/issues/1113

**Comment by 27136773** [comment]
Showing all mangas from all synchronized websites would kill the application with the current UI framework => **rejected**.
You may consider to use the [(2) Manga Search across websites](https://github.com/manga-download/hakuneko/wiki/Quick-Reference#source-selection) instead.

Tag based filtering can be implemented with less effort => **accepted**

**Comment by hypoflex** [comment]
Talked about this type of feature before with a user in #535 
see my suggestion back then here: 
https://github.com/manga-download/hakuneko/issues/535#issuecomment-544678387
